### PR TITLE
Fix 'key' warnings

### PR DESCRIPTION
**Pull request by kendallstrautman**
Some of the 'key' warnings need to be solved in Tina. But I fixed the ones I could in the toolbar.

### CODE REVIEW

File: components/cms/Toolbar.tsx [hunk 1]
```diff
@@ -27,15 +27,25 @@ export const Toolbar = styled(({ ...styleProps }) => {
...
+              const id = `${i}-${git.name}`
+              return git.component(id)
+            })}
         </Github>
         <Status>
           {status.all().length > 0 &&
-            status.all().map((status: any) => status.component())}
+            status.all().map((status: any, i) => {
+              const id = `${i}-${status.name}`
+              return status.component(id)
```
Comment (ncphillips) [inline components/cms/Toolbar.tsx]: Maybe try this:
```suggestion
              return <status.component key={id} />
```
then the plugin components don't need to set the key themselves

Comment (kendallstrautman) [reply components/cms/Toolbar.tsx]: nice!

Comment (kendallstrautman) [reply components/cms/Toolbar.tsx]: Thank you :)

File: components/cms/Toolbar.tsx [hunk 2]
```diff
@@ -27,15 +27,26 @@ export const Toolbar = styled(({ ...styleProps }) => {
           <CreateContentMenu />
         </Create>
         <Github>
-          {git.all().length > 0 && git.all().map((git: any) => git.component())}
+          {git.all().length > 0 &&
+            git
+              .all()
+              .map((git: any, i) => <git.component key={`${i}-${git.name}`} />)}
```
Comment (ncphillips) [inline components/cms/Toolbar.tsx]: On second look I would also do this:

```suggestion
              .map((git: any, i) => <git.component key={git.name} />)}
```

Using the array index in the `key` is [generally not advised](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318)

Also, `name` is unique across plugins so you'll never have two plugins of the same type with the same.

Comment (kendallstrautman) [reply components/cms/Toolbar.tsx]: okay, I wasn't sure if there would be multiple instances of a single plugin

**Pull request merged**